src/main/java/numPeopleCheck.java

```diff
@@ -0,0 +1,42 @@
+import java.util.Scanner;
+
+public class numPeopleCheck {// класс, который проверяет количество человек, на которых нужно разделить счет
```

Comment:
🍏 В Java принято именовать классы в CamelCase.

```suggestion
public class NumPeopleCheck {
```